### PR TITLE
Changes battlemage armor and wizard's hardsuit.

### PR DESCRIPTION
**Pull request by Takahiru**
This changes the battlemage armor and the wizard's hardsuit to have better specialization.

**The battlemages armor**
    It has much higher defense values, but the space protection has been taken away. 

**The wizard's hardsuit**
    It has lower defense values, and it has no slowdown, along with the space protection it had already.

This changes both to be more specialized, the battlemage armor is more combat focused, while the wizard's hardsuit is for avoiding the vacuum of space.

:cl:  
tweak: Changes the battlemage armour and the wizards hardsuit. 

/:cl:

### CODE REVIEW

**Comment by overlord12345** [comment]
the madman did it

File: code/modules/clothing/suits/wiz_robe.dm [hunk 1]
```diff
@@ -183,7 +183,7 @@
 
 /obj/item/clothing/suit/space/hardsuit/shielded/wizard
```
Comment (alexkar598) [inline code/modules/clothing/suits/wiz_robe.dm]: This is going to need repathing to not be a suit/space/hardsuit subtype

Comment (AsV9) [reply code/modules/clothing/suits/wiz_robe.dm]: @Takahiru

**Comment by alexkar598** [comment]
???
Screenshot_20200115-060318_Firefox Nightly

**Comment by Takahiru** [comment]
wait wtf did i do that wrong

**Pull request merged**